src/styles/main.scss

```diff
@@ -1,3 +1,27 @@
+@import "./blocks/header.scss";
+@import "./blocks/cards.scss";
+@import "./blocks/stars.scss";
+
+:root {
+  --card-main: #060b35;
+  --card-fallback: #616070;
+  --card-secondary: #00acdc;
+  --card-border: #f3f3f3;
+}
```

Comment:
You should use sass. Create a new folder called utils and inside it a new file called variables and declare your variables there.

$variable-name: value;